run.sh

```diff
@@ -31,36 +31,41 @@ if [ ! -d "$WORK/miniconda3" ]; then
   echo "Miniconda not found in $WORK..."
   echo "Installing..."
   mkdir -p $WORK/miniconda3
-  wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O $WORK/miniconda3/miniconda.sh
+  curl https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -o $WORK/miniconda3/miniconda.sh
   bash $WORK/miniconda3/miniconda.sh -b -u -p $WORK/miniconda3
   rm -rf $WORK/miniconda3/miniconda.sh
 
   echo "Ensuring conda base environment is OFF..."
   conda config --set auto_activate_base false
 fi
 
-echo "Initializing conda..."
-$WORK/miniconda3/bin/conda init bash
+export PATH="$WORK/miniconda3/bin:$PATH"
```

Comment:
This line is updating the PATH variable to include the directory path to the bin directory of a Miniconda installation. This is useful for making the Conda executable and associated tools available for use in the shell without needing to specify the full path each time.